nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
To confirm, do you mean: https://github.com/nest/nest-simulator/pull/650/commits/669dc4960566e289119eae6495cec1d365df6b76?diff=split#diff-06fecbcc67899362e4c211274b21fb71L791 ?
If so, that code is repeated, once after the loop, and once inside the loop:  https://github.com/nest/nest-simulator/pull/650/commits/669dc4960566e289119eae6495cec1d365df6b76?diff=split#diff-06fecbcc67899362e4c211274b21fb71L791